pkg/apis/performanceprofile/v2/performanceprofile_types.go

```diff
@@ -114,6 +114,11 @@ type CPU struct {
 	// Offline defines a set of CPUs that will be unused and set offline
 	// +optional
 	Offlined *CPUSet `json:"offlined,omitempty"`
+	// Shared defines a set of CPUs that will be shared among guaranteed workloads
+	// that needs additional cpus which are not exclusive,
+	// alongside the isolated, exclusive resources that are being used already by those workloads.
+	// +optional
+	Shared *CPUSet `json:"shared,omitempty"`
```

Comment:
Hmm, we are introducing the shared field here but the previous commit already uses it. I think we should reorder the commits maybe?

Reply:
There was a huge mess with the first commit you're absolutely right. I fixed it. Thanks!